tests/unit/model/native-access-arrays-test.js

```diff
@@ -99,17 +99,6 @@ if (CUSTOM_MODEL_CLASS) {
       assert.equal(slicedChapters[0].get('name'), 'The Boy Who Lived', `Array.slice still works`);
```

Comment:
why do we use `get` in these tests?

Reply:
To not depend on `useNativeProperties` being true

Reply:
do we test the common case of using both of these?

Reply:
The tests added here use them together.

Reply:
it's just `'mutating reference arrays cause re-renders'` isn't it?  the other two use `get`.

I guess it's probably okay, I don't have any particular reason to think these two features don't work well together, I just find it a bit odd that we don't have many tests for the most common scenario

Reply:
Happy to add more separately.